src/unum/unum.c

```diff
@@ -956,6 +971,27 @@ int main(int argc, char *argv[])
         exit(EXIT_FAILURE);
     }
 
+    // Try to read logs dir from LOG_FILES_DIR_CONSTANT
+    if(((int) util_file_to_buf(LOGS_PREFIX_FILE, logs_dir, sizeof(logs_dir) - 2)) > 0)
+    {
+        // util_fil_to_buf seems to be appending \n at the end .
```

Comment:
it's likely because you do usually press enter when adding a line to a file...
anyway this will go away when we use config instead of that file, no need to change...

BTW, it would be nice to have a util function to strip characters from specified set from the end or beginning of a string if/when we need something like what you are doing here.

Reply:
Sure. I will open a ticket to have this util function.